sphinx/source/docs/user_guide/server.rst

```diff
@@ -1352,74 +1331,73 @@ and
+is the URL of the page that hosts your Bokeh content.
 
-Now, consider when a Bokeh server is embedded inside another web page, using
-|server_document| or |server_session|. In this instance, the "Origin" header
-for the request to the Bokeh server is the URL of the page that has the Bokeh
-content embedded it. For example, if a user navigates to our page at
-``https://acme.com/products``, which has a Bokeh application embedded in it,
-then the origin header reported by the browser will be ``acme.com``. In this
-instance, we typically want to restrict the Bokeh server to honoring *only*
-requests that originate from our ``acme.com`` page, so that other pages cannot
-embed our Bokeh app without our knowledge.
+For example, if a user navigates to your page at ``https://acme.com/products``,
+the origin header reported by the browser will be ``acme.com``. In this case,
+you'd typically restrict the Bokeh server to honoring *only* the requests that
+originate from the ``acme.com`` page, preventing other pages from embedding
+your Bokeh app without your knowledge.
 
-This can be accomplished by setting the ``--allow-websocket-origin`` command
-line argument:
+You can do so by setting the ``--allow-websocket-origin`` command line argument
+as follows:
 
 .. code-block:: sh
 
     bokeh serve --show --allow-websocket-origin=acme:com myapp.py
 
-This will prevent other sites from embedding our Bokeh application in their
-pages, because requests from users viewing those pages will report a different
-origin than ``acme.com``, and the Bokeh server will reject them.
+This will prevent other sites from embedding your Bokeh application in their
+pages because requests from users viewing those pages will report a different
+origin than ``acme.com``, causing the Bokeh server to reject them.
 
 .. warning::
-    Bear in mind that this only prevents *other web pages* from surreptitiously
-    embedding our Bokeh app to an audience using standard web browsers. A
-    determined and knowledgeable attacker can spoof Origin headers.
+    Bear in mind that this only prevents *other web pages* from sneaking off
+    with your Bokeh app and displaying it to an audience using standard web
+    browsers. A determined and knowledgeable attacker can spoof Origin headers.
 
-If multiple allowed origins are required, then multiple instances of
-``--allow-websocket-origin`` can be passed on the command line.
+If you require multiple allowed origins, you can pass multiple instances of
+``--allow-websocket-origin`` on the command line.
 
-It is also possible to configure a Bokeh server to allow any and all connections
-regardless of origin:
+You can also configure the Bokeh server to allow all connections regardless of
+origin:
 
 .. code-block:: sh
 
     bokeh serve --show --allow-websocket-origin='*' myapp.py
 
-This is not recommended outside testing and experimentation.
+This option should only be used for testing and experimentation.
```

Comment:
I'd like to avoid a "should" here. Our advice is to use this setting only for testing, experimentation, or local notebook usage. (However best to express that).